File: 01_Basico/01 - Conceitos_Basicos/Respostas_em_Python/questao10.py
import math
tamanho_metros = float(input("Digite o tamanho em metros quadrados: "))
valorLata = 80.0
quantidadeLitros = tamanho_metros  / 3
latas = quantidadeLitros / 18.0
valor = math.ceil(latas) * 80
print(f"A quantidade usada sera de:    {latas:.2f} latas")
print(f"Você vai precisar comprar:     {math.ceil(latas)} latas")
print(f"O valor gasto sera:            R${valor} reais")

File: 01_Basico/01 - Conceitos_Basicos/Respostas_em_JavaScript/questao10.js
let tamanhoMetros = parseFloat(prompt("Digite o tamanho em metros quadrados: "))
let valorLata = 80.0
let quantidadeLitros = tamanhoMetros  / 3
let latas = quantidadeLitros / 18.0
let valor = Math.ceil(latas) * 80
console.log(`A quantidade usada sera de:    ${latas.toFixed(2)} latas`)
console.log(`Você vai precisar comprar:     ${Math.ceil(latas)} latas`)
console.log(`O valor gasto sera:            R$${valor} reais`)